src/data/full-quiz.js

```diff
@@ -5813,19 +5813,19 @@ const fullQuiz = [
   {
     Question: "What is the value of x ? let x = undefined || 1;",
     Answer: "1",
-    Distractor1: "1",
-    Distractor2: "undefined",
+    Distractor1: "2",
+    Distractor2: "1",

```

Comment:
None of the distractors should have a correct answer.
Only the answer field should have the correct answer.

Please change back to Distractor2: "undefined",

Reply:
Okay, I got it. Thank you 👍